__tests__/vendingmachine.spec.js

```diff
@@ -0,0 +1,199 @@
+const vendingMachine = require('../src/vendingMachine');
+// const insertedCoins = require('../src/vendingMachine');
+const inventory = require('../src/vendingMachine');
+// const coin = require('../src/vendingMachine');
+const insertedCoins = [
+  {
+    type: 'nickel',
+    value: .05,
+    stock: 0
+  },
+  {
+    type: 'dime',
+    value: .10,
+    stock: 0
+  },
+  {
+    type: 'quarter',
+    value: .25,
+    stock: 0
+  },
+  {
+    type: 'looney',
+    value: 1,
+    stock: 0
+  },
+  {
+    type:'tooney',
+    value: 2,
+    stock: 1
+  }
+];
+
+const lowCoins = [
+  {
+    type: 'nickel',
+    value: .05,
+    stock: 0
+  },
+  {
+    type: 'dime',
+    value: .10,
+    stock: 0
+  },
+  {
+    type: 'quarter',
+    value: .25,
+    stock: 0
+  },
+  {
+    type: 'looney',
+    value: 1,
+    stock: 1
+  },
+  {
+    type:'tooney',
+    value: 2,
+    stock: 0
+  }
+];
+
+const highCoins = [
+  {
+    type: 'nickel',
+    value: .05,
+    stock: 10
+  },
+  {
+    type: 'dime',
+    value: .10,
+    stock: 4
+  },
+  {
+    type: 'quarter',
+    value: .25,
+    stock: 0
+  },
+  {
+    type: 'looney',
+    value: 1,
+    stock: 1
+  },
+  {
+    type:'tooney',
+    value: 2,
+    stock: 0
+  }
+];
+
+
+
+const coin = [
+  {
+    type: 'nickel',
+    value: .05,
+    stock: 40
+  },{
+    type: 'dime',
+    value: .10,
+    stock: 48
+  },
+  {
+    type: 'quarter',
+    value: .25,
+    stock: 35
+  },
+  {
+    type: 'looney',
+    value: 1,
+    stock: 47
+  },
+  {
+    type:'tooney',
+    value:2,
+    stock: 70
+  }
+];
+
+const returnedCoins = [
+   {
+    type: 'nickel',
+    value: .05,
+    stock: 0
+  },
+  {
+    type: 'dime',
+    value: .10,
+    stock: 0
+  },
+  {
+    type: 'quarter',
+    value: .25,
+    stock: 2
+  },
+  {
+    type: 'looney',
+    value: 1,
+    stock: 0
+  },
+  {
+    type:'tooney',
+    value:2,
+    stock: 0
+  },
+];
+describe('vendingMachine', () => {
+  beforeEach(() => {
+    machine = new vendingMachine();
+  })
+  describe('When printing out inventory', () => {
+    it('should return printed inventory', () => {
+      expect(machine.printInventory()).toMatchObject(machine.printInventory());
```

Comment:
Hm. This test is testing a function call against the same function call; effectively it's doing nothing. Make sure you have an object in your test to test that call against.